raster-foundry/raster-foundry
# Enforce pro permissions for including COG in STAC export

## Overview
Enforce pro permissions for including COG in STAC export

### Checklist


## Testing Instructions
* Reassemble `api` project and restart server
* Log into Groundwork as a regular (non-pro) user
* Get the JWT and export it as `JWT_AUTH_TOKEN`
* Grab the id of a campaign you have access to
* Make a request like so, filling in your campaign id as appropriate:
```
$ echo '{
    "name": "sample export",
    "taskStatuses": [],
    "license": {
        "license": "proprietary",
        "url": null
    },
    "exportAssetTypes": ["cog"],
    "campaignId": "f851ec19-ced3-40cb-b815-9ab7552a41e9"
}' | http --auth-type=jwt POST :9100/api/stac/
```
* This request should fail with a 403
* Log into Auth0 dev user management: https://manage.auth0.com/dashboard/us/raster-foundry-dev/users/
* Edit your app metadata so it looks like this and save:
```
{
  "groundworkProUser": true
}
```
* Log out of Groundwork and log back with that user
* Copy the new JWT and export it
* Make the same request as before and it should now succeed

Closes #5623
